package.json

```diff
@@ -6,6 +6,23 @@
   "engines": {
     "vscode": "^1.68.0"
   },
+  "activationEvents": [
+    "onCommand:temporal-debugger-plugin.start",
+    "onCommand:temporal-debugger.history"
+  ],
+  "main": "./out/extension.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "temporal-debugger-plugin.start",
+        "title": "Launch Temporal Debugger Plugin"
+      },
+      {
+        "command": "temporal-debugger.history",
+        "title": "Launch Temporal Debugger History"
+      }
```

Comment:
I'm assuming this will be removed eventually... we probably only need one command. Am I missing something? Is this used for development?